android/src/main/assets/index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+<head>
+  <link href="css/style.css" rel="stylesheet"/>
+  <script src="js/jquery-3.7.1.js" type="text/javascript"></script>
```

Comment:
I don't see anywhere jquery is actually used in the HTML files

Reply:
It's not, I had high hopes of making nice pretty reactive upload screens but then remembered I can't UI a paper bag